src/util/errors-ts.ts

```diff
@@ -1072,3 +1072,30 @@ export class MissingDotenvVarsError extends NowError<
     });
   }
 }
+
+export class UnauthorizedRequestError extends NowError<
```

Comment:
rename to `UnauthorizedCertsRequestError` to help distinguish from our massive list of potential errors 😄